docs/guides/direct-managed.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Managed Resources
+toc: true
+weight: 250
+indent: true
+---
+
+# Using Managed Resources Directly
```

Comment:
I think we need a little more detail around _why_ managed resources are cluster scoped and in which cases we expect folks to use them directly. To the best of my understanding:

* We expect that application operators will _always_ consume infra via an XR and a claim.
* The only case in which we expect someone might create infra directly would be an infra operator creating supporting infrastructure like a VPC at the cluster scope.

I worry that if we keep this guide without that context folks are going to be confused about why our resources are cluster scoped. There may be some content we can copy and paste from https://blog.crossplane.io/crossplane-vs-cloud-infrastructure-addons/ to help explain this.